.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,162 @@
+name: Build Pull Request
+on:
+  - pull_request 
+env:
+  ARCH: amd64
+  TAG: v2.8-${{ github.sha }}
+  GIT_TAG: v2.8-${{ github.sha }}
+  HEAD_TAG: v2.8-${{ github.sha }}
+  COMMIT: ${{ github.sha }}
+  REPOSITORY_OWNER: ${{ github.repository_owner }}
+  IMAGE: ${{ github.repository_owner }}/rancher
+  IMAGE_AGENT: ${{ github.repository_owner }}/rancher-agent
+  IMAGE_INSTALLER: ${{ github.repository_owner }}/system-agent-installer-rancher
+  CATTLE_KDM_BRANCH: dev-v2.8
```

Comment:
I think it could be a problem because KDM could potentially need a different branch name, and we don't expect to ever need to change the references above besides once, when porting this PR to `v2.9` and `v2.7`

Reply:
Understood ^